docs/pages/inboxes/understand-sync.mdx

```diff
@@ -0,0 +1,130 @@
+
+**Apps built with XMTP aren't like mainstream messaging apps—and that can be a good thing**
+
+Most mainstream messaging apps store and sync user messaging data on centralized servers, which offers seamless, cross-device sync. But that convenience comes with a cost: user data is often retained in ways that can compromise privacy and security.
+
+XMTP, on the other hand, is built on **Messaging Layer Security (MLS)** with **forward secrecy** and **post-compromise security**. These cryptographic guarantees give users, not platforms, control of their messaging data. But this powerful shift in control doesn't come without changes to how a user's conversations, messages, and consent preferences can sync across their devices and apps.
+
+This means that your app built with XMTP may not be able to behave like a typical messaging app in every way—but that’s a feature, not a bug. With XMTP, you can build highly secure apps where users control the intentional and explicit syncing of their messaging data.
+
+This guide covers best practices for achieving the best UX for syncing user messaging data while upholding the security guarantees provided by XMTP.
+
+## 🎥 walkthrough: Syncing
+
+This video provides a walkthrough of key concepts required to implement syncing correctly.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/jl7P0onApxw?si=YNafIHebx9Kxycos" title="YouTube video player" frameBorder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerPolicy="strict-origin-when-cross-origin" allowFullScreen></iframe>
+
+## Sync messaging data history across devices and apps
+
+**I use an XMTP app on my phone. When I log into the same XMTP app on my tablet, I want to see all of my conversations, messages, and consent preferences there, too.**
+
+You can achieve this user experience using [history sync](/inboxes/history-sync) and by prompting the user for explicit actions in your app UI.
+
+History sync is enabled by default. 
+
+History sync requires that both devices (phone and tablet in this case) be online to enable the user's conversations, messages, and consent to be synced across devices. 
+
+To guide the user through using their devices to perform the history sync, you can implement a UX like the following:
+
+1. In the app on the requesting device (tablet, in this case), give the user a way to request an import of their messaging data history.
+
+   <div>
+   <img src="https://raw.githubusercontent.com/xmtp/docs-xmtp-org/refs/heads/main/docs/pages/img/requestor.png" width="300px" />
+   </div>
+
+2. Give the user a way to verify that they want to perform the import.
+
+   <div>
+   <img src="https://raw.githubusercontent.com/xmtp/docs-xmtp-org/refs/heads/main/docs/pages/img/confirm-request.png" width="300px" />
+   </div>
+
+3. In the app on the providing device (phone, in this case), notify the user of the history import request.
```

Comment:
I think the UX mockup here is incorrect and should not show no convos -- it should have the convos the requestor want to import...